### PR TITLE
fix: update inlang `settings.json`

### PR DESCRIPTION
**Pull request by NilsJacobsen**
> I'm in the core Team of inlang

## Fix inlang's configuration file

We changed the `project.inlang.json` to a `settings.json` in a `project.inlang` dir. This enables the inlang project to store app information or metadata in a dir that is not relevant to you as an end user. 

#### Old
```tree
├── project.inlang.json
```

#### New
```tree
├── project.inlang
│   ├── settings.json
```

### CODE REVIEW

File: project.inlang/project_id [hunk 1]
```diff
@@ -0,0 +1 @@
+f2fb93bb94a8ecf19f03f37f743e05cf2458b21c987774763b6725a0703a502c
```
Comment (Udit-takkar) [inline project.inlang/project_id:1]: where did you get the project id? do we need this?

Comment (NilsJacobsen) [reply project.inlang/project_id:1]: The project id is generated by the inlang sdk. It's needed to uniquely identify projects for monorepo setups.

**Comment by samuelstroschein** [comment]
@NilsJacobsen I would remove the old `project.inlang.json` file in the repo

**Comment by NilsJacobsen** [comment]
> @NilsJacobsen I would remove the old project.inlang.json file in the repo

I deleted it in my fork. But you are right, it is not displayed as a change here. @Udit-takkar if it's still there after merge you can just delete it.

**Comment by keithwillcode** [comment]
> > @NilsJacobsen I would remove the old project.inlang.json file in the repo
> 
> I deleted it in my fork. But you are right, it is not displayed as a change here. @Udit-takkar if it's still there after merge you can just delete it.

It’s showing as a renamed file here

**Comment by NilsJacobsen** [comment]
> It’s showing as a renamed file here

Ah sorry you are right. It's been moved to the new dir, that's why I do not see it as deleted. Then everything is fine.

**Review by zomars — APPROVED** [review]

**Pull request merged**